facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -1228,23 +1339,15 @@ code_behaviors <- function(input_data, wave) {
   if ("C11" %in% names(input_data)) {
     # Had "direct contact" with someone COVID-positive in the last 24 hours
     # Coded as 1 = Yes, 2 = No
-    input_data$c_direct_contact_covid <- case_when(
-      input_data$C11 == 1 ~ 1,
-      input_data$C11 == 2 ~ 0,
-      TRUE ~ NA_real_
-    )
+    input_data$c_direct_contact_covid <- input_data$C11 == 1
   } else {
     input_data$c_direct_contact_covid <- NA_real_
   }
 
-  if (all(c("C11", "C12") %in% names(input_data))) {
+  if ("C12" %in% names(input_data)) {
     # C12: was the person in C11 a member of your household
     # Coded as 1 = Yes, 2 = No
-    input_data$c_direct_contact_covid_hh <- case_when(
-      input_data$C11 == 1 & input_data$C12 == 1 ~ 1,
-      input_data$C11 == 2 | input_data$C12 == 2 ~ 0,
-      TRUE ~ NA_real_
-    )
+    input_data$c_direct_contact_covid_hh <- input_data$C12 == 1
```

Comment:
This changes the meaning of this variable, from "did they have close contact with a sick person in their household?" to "of those who had close contact, what percentage had it with someone in their household?" Is that intentional?

Reply:
There was some back and forth with FB on this item, hence the different definitions. We settled on the previous one so I'll revert this change.